File: frameworks/Java/fair/java-kora1-jdbc-repository/src/main/java/io/techempower/benchmark/kora/model/World.java
package io.techempower.benchmark.kora.model;

import ru.tinkoff.kora.database.common.annotation.Column;
import ru.tinkoff.kora.database.common.annotation.Id;
import ru.tinkoff.kora.database.common.annotation.Table;
import ru.tinkoff.kora.database.jdbc.EntityJdbc;
import ru.tinkoff.kora.json.common.annotation.Json;

@Json
@EntityJdbc
public record World(@Id @Column("id") int id,
                    @Column("randomnumber") int randomNumber) {
}

File: frameworks/Java/fair/kotlin-kora1-vertx-repository-suspend/src/main/kotlin/io/techempower/benchmark/kora/model/World.kt
package io.techempower.benchmark.kora.model

import ru.tinkoff.kora.database.common.annotation.Column
import ru.tinkoff.kora.database.common.annotation.Id
import ru.tinkoff.kora.json.common.annotation.Json

@Json
data class World(
    @field:Id @field:Column("id") val id: Int,
    @field:Column("randomnumber") val randomNumber: Int,
)
